### PR TITLE
kubevirt, hcp, HOSTEDCP-1311: Add multinet knobs

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
The main feature to configure multi network at kubevirt provider is in place but the knobs are hcp cli are missing. This change duplicate the flags from hypershift cli to hcp related to multinet.

### CODE REVIEW

File: product-cli/cmd/cluster/kubevirt/create.go [hunk 1]
```diff
@@ -41,6 +45,8 @@ func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
...
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.NetworkInterfaceMultiQueue, "network-multiqueue", opts.KubevirtPlatform.NetworkInterfaceMultiQueue, `If "Enable", virtual network interfaces configured with a virtio bus will also enable the vhost multiqueue feature for network devices. supported values are "Enable" and "Disable"; default = "Disable"`)
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.QoSClass, "qos-class", opts.KubevirtPlatform.QoSClass, `If "Guaranteed", set the limit cpu and memory of the VirtualMachineInstance, to be the same as the requested cpu and memory; supported values: "Burstable" and "Guaranteed"`)
+	cmd.Flags().StringArrayVar(&opts.KubevirtPlatform.AdditionalNetworks, "additional-network", opts.KubevirtPlatform.AdditionalNetworks, fmt.Sprintf(`Specify additional network that should be attached to the nodes, the "name" field should point to a multus network attachment definition with the format "[namespace]/[name]", Supported parameters: %s, example: "name:ns1/nad-foo`, params.Supported(kubevirt.NetworkOpts{})))
```
Comment (orenc1) [inline product-cli/cmd/cluster/kubevirt/create.go:48]: the type of this flag is StringArrayVar, so i'm assuming multiple additional networks can be defined. in that case:

1. the name of the flag is singular (`additional-network`), not plural.
2. i think you should give an example of multiple entries in this arg.

[the remark this answers is not in the document]
Comment (qinqon) [reply product-cli/cmd/cluster/kubevirt/create.go:48]: Yes that's ok, it will be an empty slice.

Comment (orenc1) [reply product-cli/cmd/cluster/kubevirt/create.go:48]: then maybe you can add something like `Can be specified multiple times.` as we have for `--annotations` for clarity?

**Review by davidvossel — COMMENTED** [review]
/approve

**Review by orenc1 — APPROVED** [review]
/lgtm

**Pull request merged**